test/unittest/unittest_nntrainer_internal.cpp

```diff
@@ -46,10 +119,178 @@ TEST(nntrainer_NeuralNetwork, setConfig_02_n) {
+  int status = ML_ERROR_NONE;
+  nntrainer::DataBuffer data_buffer;
+  status = data_buffer.setClassNum(0);
+  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
+}
+
+/**
+ * @brief Data Buffer set train Data file
+ */
+TEST(nntrainer_DataBuffer, setDataFile_01_p) {
+  int status = ML_ERROR_NONE;
+  nntrainer::DataBuffer data_buffer;
+  status = data_buffer.setDataFile("./data/trainingSet.dat", nntrainer::DATA_TRAIN);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+}
+
+/**
+ * @brief Data Buffer set train Data file
+ */
+TEST(nntrainer_DataBuffer, setDataFile_02_n) {
+  int status = ML_ERROR_NONE;
+  nntrainer::DataBuffer data_buffer;
+  status = data_buffer.setDataFile("./data/no_exist.dat", nntrainer::DATA_TRAIN);
+  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
+}
+
+/**
+ * @brief Data Buffer set train Data file
+ */
+TEST(nntrainer_DataBuffer, setDataFile_03_p) {
+  int status = ML_ERROR_NONE;
+  nntrainer::DataBuffer data_buffer;
+  status = data_buffer.setClassNum(10);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+  status = data_buffer.setDataFile("./data/label.dat", nntrainer::DATA_LABEL);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+}
+
+/**
+ * @brief Data Buffer set train Data file
+ */
+TEST(nntrainer_DataBuffer, setDataFile_04_n) {
+  int status = ML_ERROR_NONE;
+  nntrainer::DataBuffer data_buffer;
+  status = data_buffer.setClassNum(3);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+  status = data_buffer.setDataFile("./data/label.dat", nntrainer::DATA_LABEL);
+  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
+}
+
+/**
+ * @brief Optimizer set type
+ */
+TEST(nntrainer_Optimizer, setType_01_p) {
+  int status = ML_ERROR_NONE;
+  nntrainer::Optimizer op;
+  nntrainer::OptType t = nntrainer::OptType::adam;
+  status = op.setType(t);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+}
+
+/**
+ * @brief Optimizer set type
+ */
+TEST(nntrainer_Optimizer, setType_02_p) {
+  int status = ML_ERROR_NONE;
+  nntrainer::Optimizer op;
+  nntrainer::OptType t = nntrainer::OptType::sgd;
+  status = op.setType(t);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+}
+
+/**
+ * @brief Optimizer set type
+ */
+TEST(nntrainer_Optimizer, setType_03_n) {
+  int status = ML_ERROR_NONE;
+  nntrainer::Optimizer op;
+  nntrainer::OptType t = nntrainer::OptType::unknown;
+  status = op.setType(t);
+  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
+}
+
+/**
+ * @brief Optimizer set Opt Param
+ */
+TEST(nntrainer_Optimizer, setOptParam_01_p) {
+  int status = ML_ERROR_NONE;
+  nntrainer::Optimizer op;
+  nntrainer::OptType t = nntrainer::OptType::adam;
+  nntrainer::OptParam p;
+  status = op.setType(t);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+  p.learning_rate = -0.001;
+  p.beta1 = 0.9;
+  p.beta2 = 0.9999;
+  p.epsilon = 1e-7;
+  p.weight_decay.type = nntrainer::WeightDecayType::l2norm;
+  p.weight_decay.lambda = 0.001;
+  status = op.setOptParam(p);
+  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
+}
+
 /**
  * @brief Main gtest
  */
-int main(int argc, char **argv) {
+int main(int argc, char** argv) {
```

Comment:
Any reason?

Reply:
Cause we are using clang-format-4.0. It changes automatically.